src/test/java/com/github/redouane59/twitter/nrt/TwitterClientTest.java

```diff
@@ -253,6 +254,13 @@ public void testLikeTweet(){
         assertEquals("1107533", unlikedTweet.getId());
     }
 
+    @Test
+    public void testPostTweet(){
+        String text = "API Test " + LocalDateTime.now() + " #TwitterAPI";
+        ITweet result = twitterClient.postTweet(text);
+        assertNotNull(result);
```

Comment:
Could you test more properties of this `result`?

Reply:
Yes why not, I'm adding on another PR : 
``` 
        assertNotNull(result.getId());
        assertEquals(text,result.getText());
```